### PR TITLE
added LogIN Databse

### PR DESCRIPTION
**Pull request by anelej007**
- Added linkage in sign up process to be able to create DB entry upon successful signup
- Fixed Fitbit username DB insertion
- Added LogIn checking Database content process
- Tested sunny and rainy day scenario. Entries get created in DB and can be correctly checked upon log in.

### CODE REVIEW

File: controllers/doctors_HISP.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿from bottle import route, view, request, response
 from datetime import datetime
 import sqlite3
+from permissions import permissions
```
Comment (omtinez) [inline controllers/doctors_HISP.py]: What is this for? Where is it used?

**Comment by anelej007** [comment]
i was not able to run app on my end without importing modules. In this specific case for example, I got errors complaining about permissions being unknown.

**Comment by omtinez** [comment]
I see, good catch. Probably my mistake during the refactoring. Merged.

**Comment by anelej007** [comment]
thank you

**Pull request merged**